src/Parse/ParseUser.php

```diff
@@ -157,6 +157,33 @@ public static function logIn($username, $password)
         return $user;
     }
 
+    /**
+     * Uses the master key to log in and return a valid ParseUser, or throws if invalid.
+     *
+     * @param $userId
+     *
+     * @throws ParseException
+     *
+     * @return static
```

Comment:
`@return ParseUser` to improve documentation

Reply:
Done